### PR TITLE
Add local task for notifying Sentry

### PR DESCRIPTION
**Pull request by pspanja**
The bug was fixed in https://github.com/deployphp/recipes/pull/224, but since the release is limited to PHP `~7.0` and the repo is archived, there is no other option than to redefine the task locally, at least until v7 is released.

### CODE REVIEW

File: deploy/tasks/sentry_notify.php [hunk 1]
```diff
@@ -0,0 +1,186 @@
+<?php
```
Comment (iherak) [inline deploy/tasks/sentry_notify.php:1]: Why not keep this in the already existing `deploy/tasks/sentry.php` file?

Comment (iherak) [reply deploy/tasks/sentry_notify.php:1]: Actually, even better, in `overrides.php` file which was meant for overrides of existing tasks from recipes :)

File: deploy.php [hunk 2]
```diff
@@ -14,6 +14,7 @@
 require __DIR__ . '/deploy/tasks/rsync.php';
 require __DIR__ . '/deploy/tasks/deploy_log.php';
 require __DIR__ . '/deploy/tasks/sentry.php';
++require __DIR__ . '/deploy/tasks/sentry_notify.php';
```
Comment (emodric) [inline deploy.php:17]: You have an extra `+` here.

Comment (pspanja) [reply deploy.php]: Fixed in https://github.com/netgen/media-site/pull/90/commits/e99d654124911bf2a2371280a48559b302112e53

File: deploy/tasks/sentry_notify.php (hunk 1 above)
Comment (pspanja) [reply deploy/tasks/sentry_notify.php]: Updated in https://github.com/netgen/media-site/pull/90/commits/b16cc52a0abc7bd681bbb73fbe3f58f630e77724

**Pull request closed without merging**